doxygen.yaml

```diff
@@ -0,0 +1,10 @@
+name: doxygen
+buildsystem: cmake-ninja
+builddir: true
+sources:
+  - type: archive
+    url: https://github.com/doxygen/doxygen/archive/Release_1_8_15.tar.gz
+    sha256: cc5492b3e2d1801ae823c88e0e7a38caee61a42303587e987142fe9b68a43078
+cleanup: ['*']
+modules:
+  - graphviz.yaml
```

Comment:
I'd prefer if modules were kept in one file instead if splitting the hair in two. It doesn't seem it's re-used elsewhere.